### PR TITLE
A recap blog-post for pyconug

### PR DESCRIPTION
**Pull request by TobiasHT5**
## Issue Link 🔗: https://github.com/BlackPythonDevs/blackpythondevs.github.io/issues/530

**Issue**: #530 

## Type of Change

- [ ] Bug fix 🐞
- [ ] New feature/page
- [ ] Documentation update
- [ ] Other

## Description 📋

- **What:** Wrote a blog post to recap the PyConUG conference

- **Why:** It was necessary to recap this event since it was sponsored by the Black Python Devs community

- **How:** 

## Checklist ✅

- [ ] Followed the [Code of Conduct](https://github.com/BlackPythonDevs/blackpythondevs.github.io?tab=coc-ov-file) and [Contribution Guide](https://github.com/BlackPythonDevs/blackpythondevs.github.io/blob/469afd71943738b7209bfe56d5a4e35394fbaafc/CONTRIBUTING.md)
- [x] Ran `pre-commit run --all`
- [ ] All tests pass locally
- [ ] Added tests (if applicable)
- [ ] Documentation updated (if applicable)

## Additional Notes & Screenshots

Add any additional notes or comments that might be helpful for the reviewers.

### CODE REVIEW

**Review by kjaymiller — CHANGES_REQUESTED** [review]
one small suggestion and then I think you're good to go!

**Comment by TobiasHT5** [comment]
> one small suggestion and then I think you're good to go!

What could this suggestion be @kjaymiller ?

File: _posts/2024-10-25-pycon-ug-2024.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+date: 2024-10-25
+author: "Tobias Thomas (Toe•bias HT)"
+lang: en
+featured_image: "/assets/images/pyconug.png"
+---
+
+
+Wednesday, 9th, October, 2024, was Uganda's Independence day. </br>
+</br>
+We were celebrating 62 years of independence. This also happened to be the first day of the 2nd installment of Pyconug conference 2024. It was a very exciting day and alot of the Ugandan pythonistas and pythoneers were waiting in anticipation for this day. Pyconug had alot of remarkable sponsors this year; from international brands like Github, JetBrains and the Python Software Foundation to local brands like Africa's talking platform and more. </br>
```
Comment (kjaymiller) [inline _posts/2024-10-25-pycon-ug-2024.md]:
```suggestion
We were celebrating 62 years of independence. This also happened to be the first day of the 2nd installment of [Pyconug conference 2024](https://ug.pycon.org/2024). It was a very exciting day and alot of the Ugandan pythonistas and pythoneers were waiting in anticipation for this day. Pyconug had alot of remarkable sponsors this year; from international brands like Github, JetBrains and the Python Software Foundation to local brands like Africa's talking platform and more. </br>
```

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Apologies... I thought I had created this comment. We need to add a link to the page.

File: _posts/2024-10-25-pycon-ug-2024.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: "Black Python Devs at PyconUG 2024"
+date: 2024-10-25
+author: "Tobias Thomas (Toe•bias HT)"
+lang: en
+featured_image: "/assets/images/pyconug.png"
```
Comment (kjaymiller) [inline _posts/2024-10-25-pycon-ug-2024.md]: We have an image in canva for this sponsorship that should be used as the featured image.

link: https://www.canva.com/design/DAGQkaiQC88/G4IKEN9UrDbHEbm2DZBLQA/edit?utm_content=DAGQkaiQC88&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

@ToebiasHT - let me know if you can't download it and I will get it added to the images path.

**Review by kjaymiller — CHANGES_REQUESTED** [review]

File: _posts/2024-10-25-pycon-ug-2024.md (hunk 2 above)
Comment (TobiasHT5) [reply _posts/2024-10-25-pycon-ug-2024.md]: @kjaymiller I think I've been able to incorporate it. Shall I resolve the conversation?

**Review by kjaymiller — APPROVED** [review]
Greate work on the changes!

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Looks like pre-commit isn't happy @ToebiasHT please run `pre-commit run --all` to fix. If it still errors we can troubleshoot.

**Comment by TobiasHT5** [comment]
> Looks like pre-commit isn't happy @ToebiasHT please run `pre-commit run --all` to fix. If it still errors we can troubleshoot.

@kjaymiller  I don't know where to run that command. Any instructions I could follow?

**Comment by kjaymiller** [comment]
> > Looks like pre-commit isn't happy @ToebiasHT please run `pre-commit run --all` to fix. If it still errors we can troubleshoot.
> 
> @kjaymiller I don't know where to run that command. Any instructions I could follow?

you should run it in the terminal

**Comment by TobiasHT5** [comment]
@kjaymiller I've run it, it says prettier failed. Not sure how to continue from there.
Screenshot 2024-10-29 at 12 32 25

**Comment by kjaymiller** [comment]
now commit the changes and it should have fixed the issues.

**Comment by TobiasHT5** [comment]
@kjaymiller changes have been committed. Let's give it another go.

**Review by kjaymiller — APPROVED** [review]
Okay looks good and tests are passing.

Sorry that pre-commit snagged you on that one. It's something that a lot of folks wrestle with and pre-commit at times feels like a hassel but it does help keep us from having to push too many "lint and format" PRs

**Comment by TobiasHT5** [comment]
Got it. Next time will be a breeze...

**Comment by TobiasHT5** [comment]
@kjaymiller I've just noticed that the `</br>` HTML tags are not being rendered, they are leaking through into the post.

**Comment by kjaymiller** [comment]
Submit a fix PR and we'll get it pushed through.

**Pull request merged**